MOLFCMClient/MOLFCMClient.m

```diff
@@ -31,9 +31,12 @@
 static NSString *const kFCMApplicationJSON = @"application/json";
 static NSString *const kFCMContentType = @"Content-Type";
 
-/**  15 minute backoff maximum */
+/**  Default 15 minute backoff maximum */
 static const uint32_t kBackoffMaxSeconds = 900;
 
+/**  Default 10 sec connect delay maximum */
+static const uint32_t kConnectDelayMaxSeconds = 10;
```

Comment:
These 2 consts would be better having default in the name

Reply:
Good idea